src/main/java/org/fogbowcloud/manager/core/models/orders/StorageOrder.java

```diff
@@ -1,15 +1,24 @@
 package org.fogbowcloud.manager.core.models.orders;
 
-import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
+import org.fogbowcloud.manager.core.instanceprovider.InstanceProvider;
 import org.fogbowcloud.manager.core.models.token.Token;
 
 public class StorageOrder extends Order {
 
-    private int storageSize;
+	private int storageSize;
 
-	public StorageOrder(OrderState orderState, Token localToken, Token federationToken, String requestingMember,
-						String providingMember, OrderInstance orderInstace, long fulfilledTime, int storageSize) {
-		super(orderState, localToken, federationToken, requestingMember, providingMember, orderInstace, fulfilledTime);
+	/**
+	 * Creating Order with predefined Id.
+	 */
+	public StorageOrder(String id, Token localToken, Token federationToken, String requestingMember, String providingMember,
```

Comment:
one call the other

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/f76e0578247c51bfa5f1fbb8168a77f6fa13c438).